tests/e2e/support/objects/app-files/page/shares/withMe.ts

```diff
@@ -14,6 +15,8 @@ export class WithMe {
 
   async navigate(): Promise<void> {
     await this.#page.locator(sharesNavSelector).click()
+    const a11yObject = new objects.a11y.Accessibility({ page: this.#page })
+    await a11yObject.getSevereAccessibilityViolations('#files')
```

Comment:
this will only return the violations. without assertion this will pass even if there are some serious violations